### DISCUSSION TITLE
lenship2's textures

### DISCUSSION THREAD

**Discussion by lenship2** [StrataSource/Portal-2-Community-Edition#1779] [Resources]
# **lenship2's Texture Repository**

This discussion should contain mostly every texture I've created for Portal 2, all kept inside this zip file hosted in Google Drive. Simply download the file, and extract the files to wherever you keep your custom assets folder.

https://drive.google.com/file/d/1GAnyiVvaNnZjMXSqn7PW0bOvxrpXFXD-/view?usp=sharing

I'll hopefully be updating this as I create more textures, so keep on the lookout for that ^^

If you have any feedback, please let me know! You can contact me on Discord under the username lenship2.

## **Features:**

- Parallax Mapped PBR* versions of most of the popular Portal 2 textures
- The entirety of my [BTS Alphabet](https://github.com/StrataSource/Portal-2-Community-Edition/discussions/1743)
- Parallax OBB tool texture
- Wheatley Laboratories overlay
- 1994 Aperture Science Periodic Table
- Refracting 'frosted' glass

## Shiny Textures?

While I don't personally consider it an issue, you might find that some of the textures appear too 'glossy' or 'wet' in-game. Fortunately, this is easily adjustable!
To adjust the roughness:
1. Go into the .vmt file of whatever texture you're unhappy with.
2. Go to the $mraoscale parameter, and edit the second number to be higher.
![Screenshot 2024-12-19 155906](https://github.com/user-attachments/assets/2c19ee33-7c3b-4e8a-ba47-b36a6432f4f0)
3. Test the texture in game, and increase/decrease the second number until you're satisfied.

## **Crediting**

Credits aren't required, unless used in a competition or mod. You can credit me as lenship2 in those cases.

## 

*_A majority of textures are PBR specific, and made specifically to work with Portal 2 Community Edition, so compatibility issues with base Portal 2 will arise with some of the textures._

**Comment by DerpyTheHedgeone** [comment]
link doesnt work anymore
